geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/KeywordCommand.scala

```diff
@@ -0,0 +1,99 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.kafka.commands
+
+import com.beust.jcommander._
+import com.beust.jcommander.converters.IParameterSplitter
+import org.locationtech.geomesa.kafka.KafkaDataStoreSchemaManager
+import org.locationtech.geomesa.tools.kafka.commands.KeywordCommand.KeywordParameters
+import org.locationtech.geomesa.tools.common.{FeatureTypeNameParam}
+import org.locationtech.geomesa.tools.kafka.ProducerKDSConnectionParams
+
+import scala.collection.JavaConversions._
+import scala.io.StdIn
+
+class KeywordCommand(parent: JCommander) extends CommandWithKDS(parent) {
+
+  import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType._
+  override val command: String = "keywords"
+  override val params = new KeywordParameters()
+
+  override def execute(): Unit = {
+
+    if (ds == null) {
+      throw new ParameterException("Could not load a data store with the provided parameters")
```

Comment:
Unnecessary if statement. The `new DataStoreHelper(params).getDataStore` call checks the parameters.